api/src/main/java/org/openmrs/module/pihcore/PihCoreUtil.java

```diff
@@ -37,7 +41,11 @@ public static String getDrugListChecksumDirectory() {
     }
 
     public static String getFormResource(String formName) {
-        return "file:" + getFormDirectory() + formName;
+        try {
+            return URLEncoder.encode("file:" + getFormDirectory() + formName, StandardCharsets.UTF_8.toString()).toString();
```

Comment:
Doesn't URLEncoder.encode return a string (ie, why do we need the toString at the end?)